src/ImpactX.cpp

```diff
@@ -199,6 +215,95 @@ namespace impactx {
                                        << " slice_step=" << slice_step << "\n";
                     }
 
+                    //CSR Wakefield Response
+
+                    bool element_has_csr = false; // Updates to true for example with bend element
```

Comment:
We need to generalize this, e.g., as an attribute to the elements that have CSR or as a small map/list of elements that have CSR, stored external to them.

I assume we will have multiple wakefield models for various elements later on?